cli/azd/resources/apphost/templates/resources.bicept

```diff
@@ -345,7 +346,31 @@ resource {{bicepName $name}}OperatorRoleAssignment 'Microsoft.Authorization/role
     roleDefinitionId:  subscriptionResourceId('Microsoft.Authorization/roleDefinitions', '230815da-be43-4aae-9cb4-875f7bd000aa')
   }
 }
-{{- end}}
+{{range $cname := $value.Databases}}
+resource {{bicepName $cname}} 'Microsoft.DocumentDB/databaseAccounts/sqlDatabases@2023-04-15' = {
+  name: '{{$cname}}'
+  location: location
+  tags: union(tags, {'aspire-resource-name': '{{$cname}}'})
+  parent: {{bicepName $name}}
```

Comment:
NIT: In other places we prefer to nest the definitions instead of having explicit `parent` properties to tie the nested resources to the parent.  You can see how this was done for the some of the ServiceBus stuff, I think.

This is fine for now - just making that changes makes the bicep a little more idiomatic, I think.